modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.tsx

```diff
@@ -49,9 +49,9 @@ const PolicyDetail = ({
 }: ApplicationPolicyDetailsType) => {
 	enum navBarLabels {
 		Vehicles = 'Vehicles',
-		Drives = 'Drives',
+		Drivers = 'Drivers',
```

Comment:
Can you reorder?

Drivers first, Vehicles last

Reply:
For enum, the order doesn't matter, since you are adding a value to it.

But, if you have

`
Drivers,
Vehicle
`

Maybe it matter, because the ordering will add an implicit number, under the hood, it does:

`
Drivers, = 0
Vehicle = 1
`